src/main/java/frc/robot/subsystems/base/conveyor/ConveyorBase.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems.base.conveyor;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * @author Barel
+ * @version 1.0
+ * @since 2020-off
+ * <p>
+ * The class contains various methods that every {@code Wrist} <b>should</b> implement.
+ * <p>
+ * The purpose of this class is to provide guidelines and restrict what important to us and what not.
+ */
+public abstract class ConveyorBase extends SubsystemBase {
```

Comment:
Please add method to count the amount of balls in the conveyor

Reply:
I wanted to add a Coutable interface, but I forgot, maybe this would be a better solution

Reply:
Do you need an interface for one incrementing integer?